.gitignore

```
@@ -0,0 +1,3 @@
+_cache
+*.exe
+*.xz
```

New line and end on file?
```suggestion
*.xz

```